blocks/identity-block/features/header-account-action/default.jsx

```diff
@@ -64,8 +74,57 @@ const HeaderAccountAction = ({ customFields }) => {
 
   // What do we want to happen if there is an error?
   return (
-    <div className="xpmedia-subs-header">
-      <a href={loginURL} className="xpmedia-subs-header--button">Sign In</a>
+    <div className="xpmedia-subs-header--logged-out-header">
+      <div className="xpmedia-subs-header--desktop-logged-out-header">
+        <Button
+          ariaLabel={phrases.t('identity-block.sign-up')}
+          // should be an a tag if it's a link
+          as="a"
+          buttonSize={BUTTON_SIZES.SMALL}
+          buttonStyle={BUTTON_STYLES.OUTLINED_GREY}
+          buttonType={BUTTON_TYPES.LABEL_ONLY}
+          href={createAccountURL}
+          text={phrases.t('identity-block.sign-up')}
+        />
+        <Button
+          ariaLabel={phrases.t('identity-block.log-in')}
+          // should be an a tag if it's a link
+          as="a"
+          buttonSize={BUTTON_SIZES.SMALL}
+          buttonStyle={BUTTON_STYLES.WHITE_BACKGROUND_FILLED}
+          buttonType={BUTTON_TYPES.LABEL_AND_ICON}
+          href={loginURL}
+          iconType="user"
+          text={phrases.t('identity-block.log-in')}
+        />
+      </div>
+      <div className="xpmedia-subs-header--mobile-logged-out-header">
+        <Button
+          // should be button if toggleable
+          as="button"
+          buttonSize={BUTTON_SIZES.SMALL}
+          buttonStyle={BUTTON_STYLES.WHITE_BACKGROUND_FILLED}
+          buttonType={BUTTON_TYPES.ICON_ONLY}
+          iconType="user"
+          onClick={() => setAccountMenu(!isAccountMenuOpen)}
+          // for button accessibility
+          type="button"
+        />
+        {isAccountMenuOpen && (
+          <DropdownLinkList>
+            <DropDownLinkListItem
+              href={createAccountURL}
+              text={phrases.t('identity-block.sign-up')}
+              isLastItem={false}
+            />
+            <DropDownLinkListItem
+              href={loginURL}
+              text={phrases.t('identity-block.log-in')}
+              isLastItem
```

Comment:
Instead of this flag, why aren't we just styling with `&:last-of-type {}` ?

Reply:
updated -- thanks